psydac/fem/basic.py

```diff
@@ -300,3 +341,5 @@ def __isub__(self, other):
         assert self._space is other._space
         self._coeffs -= other._coeffs
         return self
+
+from .vector import ProductFemSpace
```

Comment:
I guess that you're importing `ProductFemSpace` here at the end of the file in order to avoid issues with this cyclic dependency (vector -> basic -> vector). A better way to do this is to move this import to the methods that actually use `ProductFemSpace`, namely `__mul__` and `__rmul__`.

Reply:
done